src/app/organizer/Actions.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { ReactNode, useCallback } from 'react';
+import _ from 'lodash';
+import { t } from 'app/i18next-t';
+import styles from './Actions.m.scss';
+import { AppIcon, lockIcon } from 'app/shell/icons';
+import DropDown, { DropDownItem } from './DropDown';
+import { itemTagList, TagInfo } from 'app/inventory/dim-item-info';
+import { DimStore } from 'app/inventory/store-types';
+
+export interface ColumnStatus {
+  id: string;
+  content: ReactNode;
+  enabled: boolean;
+  subColumnIds: string[];
+}
+
+const bulkItemTags = Array.from(itemTagList);
+bulkItemTags.push({ type: 'clear', label: 'Tags.ClearTag' });
+
+function Actions({
+  stores,
+  itemsAreSelected,
+  onLock,
+  enabledColumns,
+  onChangeEnabledColumn,
+  onChangeColumnOrder,
+  onTagSelectedItems,
+  onMoveSelectedItems
+}: {
+  stores: DimStore[];
+  itemsAreSelected: boolean;
+  enabledColumns: ColumnStatus[];
+  onChangeEnabledColumn(item: ColumnStatus): void;
+  onChangeColumnOrder(columnStatus: ColumnStatus[]): void;
```

Comment:
I'm getting the feeling that this one component is doing multiple things. Maybe it's better to split out a column-chooser and item action buttons into their own components. Then, you can group them together with any other component that just takes them as children.